### PR TITLE
Update site content

### PR DESCRIPTION
**Pull request by zapbot**
From:
zaproxy/zaproxy-website@a035583bb3f3f5879779921811c76a8a03669e2a

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -1 +0,0 @@
-include: [".well-known"]
```
Comment (thc202) [inline _config.yml:1]: We need to add this to the website repo.

Comment (kingthorin) [reply _config.yml:1]: Happy to, just need to know where in the structure?

Comment (psiinon) [reply _config.yml:1]: Isnt it already there? https://github.com/zaproxy/zaproxy-website/blob/master/site/static/.well-known/security.txt

Comment (kingthorin) [reply _config.yml:1]: Ya, I was assuming he was referring to the Jekyll config file. So I PRd that.

Comment (thc202) [reply _config.yml:1]: Yes, it was the config file.

**Comment by thc202** [comment]
This is good to go now.

**Review by psiinon — APPROVED** [review]

**Pull request merged**